RCTComms/comms.py

```diff
@@ -1159,4 +1225,7 @@ def execute_cb(self, event_value: int, kwargs):
             cb_(**kwargs)
 
     def registerCallback(self, event: EVENTS, callback):
-        self.__packetMap[event.value].append(callback)
+        if event.value in self.__packetMap:
+            self.__packetMap[event.value].append(callback)
+        else:
+            self.__packetMap[event.value] = [callback]
```

Comment:
It may be better instead to instantiate `self.__packetMap` with empty lists to start with